conversion/conversion.go

```diff
@@ -433,38 +438,70 @@ func UpdateDDLForeignKeys(project, instance, dbName string, conv *internal.Conv,
 	ctx := context.Background()
 	adminClient, err := database.NewDatabaseAdminClient(ctx)
 	if err != nil {
-		return fmt.Errorf("can't create admin client: %w\n", analyzeError(err, project, instance))
+		fmt.Printf("can't create admin client: %s\n", analyzeError(err, project, instance))
+		conv.Unexpected(fmt.Sprintf("can't create admin client: %s\n", analyzeError(err, project, instance)))
+		return err
```

Comment:
what is the reason for changing the behavior here? It seems like just returning the error is the right thing to do here to be bubbled up to the end user